wls-gui-wahllokalsystem/src/composables/user/userValidator.ts

```diff
@@ -0,0 +1,38 @@
+import type { UserDTO } from "@/api/wls-clients/generated-auth-api";
+
+export function useUserValidator() {
+  function validUserDtoOrThrow(userDto: UserDTO) {
+    const requiredFields = [
```

Comment:
So meckert auch der Compiler wenn darin eine Prop enthalten ist die es nicht im UserDTO gibt. Das macht es wartbarer.

```suggestion
    const requiredFields: (keyof UserDTO)[] = [
```

Reply:
[beb6efc](https://github.com/it-at-m/Wahllokalsystem/pull/1450/commits/beb6efc8181902cbeb04d5645317a0ac874cae67)